books/RayTracingTheNextWeek.html

```diff
@@ -94,10 +93,35 @@
 </div>
 
 
+Managing Time
+--------------
+Before continuing, let's think about time, and how we might manage it across one or more renders.
+There are two aspects of shutter timing to think about: the time from one shutter opening to the
+next shutter opening, and how long the shutter stays open for each frame. Standard movie film used
+to be shot at 24 frames per second. Modern digital films can be 30 or even 60 frames per second.
+Each frame can have its own shutter speed, which need not be -- and typically isn't -- the maximum
+duration of the entire shutter period. You could have the shutter open for 1/1000th of a second per
+frame, or 1/60th of a second. It may surprise you to learn that for most film motion pictures, the
+screen is actually dark more than it is lit up with the movie frames!
+
+If you wanted to render a sequence of images, you would need to set up the camera with the
+appropriate shutter timings: frame-to-frame period, shutter/render duration, and the total number of
+frames (or total shot time). If the camera is moving and the world is static, you're good to go.
+However, if anything in the world is moving, you would need to add a method to `hittable` to
```

Comment:
"you would need to add a method to `hittable` to ..."
->
"you would need to rewrite `hittable` so that every object was aware of the time of a ray. This could be done with ..."

Reply:
Correction: every object would need to be aware of the frame period (to compute the bounding box for that frame). The _hit_ method handles ray time.

Reply:
Unfortunately, this statement has to be a bit hand-wavy, since we haven't yet introduced bounding boxes.